### PR TITLE
Fix zipping and logging

### PR DESCRIPTION
**Pull request by PepijnSenders**
# What this PR changes:
- I rewrote the tarring logic, to use the native OS `tar` function for now. Doesn't seem like any good solutions are in `node` at the moment. Will try to keep checking, one that supports huge folder compression like the good old terminal.

# When reviewing, please consider:
- After this Windows OS will be not working anymore.

### CODE REVIEW

**Comment by kieranajp** [comment]
djon djon

File: src/index.js [hunk 1]
```diff
@@ -91,7 +93,9 @@ const run = (
                 });
         })
         .then(() => {
-            log('Djon.');
+            const mushroom = getEmoji('mushroom');
+
+            log(`Djon ${mushroom}`);
```
Comment (kieranajp) [inline src/index.js]: Just:

```js
log(`Djon ${getEmoji('mushroom')}`);
```

Comment (PepijnSenders) [inline src/index.js]: Shiny 💎

Comment (kieranajp) [inline src/index.js]: 🍄

File: package.json [hunk 2]
```diff
@@ -12,6 +12,9 @@
   },
   "keywords": [],
   "author": "Pepijn Senders <ps@hellofresh.com>",
+  "contributors": [
+    "Kieran Patel <kp@hellofresh.com> (https://kieranajp.uk)"
```
Comment (PepijnSenders) [inline package.json]: 📥

**Comment by kieranajp** [comment]
:shipit:

**Pull request merged**